src/webgl/p5.RendererGL.js

```diff
@@ -375,6 +460,63 @@ p5.RendererGL.prototype._setNoFillStroke = function() {
 };
 
 /**
+ * Get a region of pixels from an image.
+ *
+ * If no params are passed, those whole image is returned,
+ * if x and y are the only params passed a single pixel is extracted
+ * if all params are passed a rectangle region is extracted and a p5.Image
+ * is returned.
+ *
+ * Returns undefined if the region is outside the bounds of the image
+ *
+ * @method get
+ * @param  {Number}               [x] x-coordinate of the pixel
+ * @param  {Number}               [y] y-coordinate of the pixel
+ * @param  {Number}               [w] width
+ * @param  {Number}               [h] height
+ * @return {Array|Color|p5.Image}     color of pixel at x,y in array format
```

Comment:
agree, we recently discussed this with the documentation for get() in 2D mode. you could take a look at that entry for guidance.